build/.goreleaser.yml

```diff
@@ -37,6 +37,8 @@ changelog:
     exclude:
      - '^docs:'
      - '^test:'
+     - Merge pull request
```

Comment:
Not quite familiar with the format used here.  Is is a regex or a prefix?

Reply:
It is regular expression. I copied this from `GoReleaser`'s config file itself :).